src/redux/hotel/hotel.js

```diff
@@ -18,8 +18,8 @@ const hotelSlice = createSlice({
 
 export const { getAll } = hotelSlice.actions;
 
-export const getAllRoomsAsync = () => async (dispatch) => {
-  const data = await getAllRooms();
+export const getAllHotelsAsync = () => async (dispatch) => {
+  const data = await getAllHotels();
   dispatch(getAll(data));
 };
```

Comment:
- [ ] Since we are working with redux toolkit we should use `createAsyncThunk` and `extraReducers` to handle side effects in our actions dispatch, otherwise, this custom function must be tested